### DISCUSSION TITLE
Hello Everyone!

### DISCUSSION THREAD

**Discussion by git900000** [nikoniko78/scripts-java-physion#31] [General]
Hi! I am not a signed-in user on Physion, but I am a good designer. Does anyone need something designed for them?

**Comment by nikoniko78** [comment]
Hello, git900000! We would be happy to have someone in the community who knows how to design. This will be extremely helpful for the community. Would you mind introducing yourself more?

-nikoniko78

**Reply by git900000** [reply]
You can call me Gitney (Yes, I made that name up; I couldn't really think of anything lol). I just spend my free time on Physion, sometimes playing around with ragdolls and liquids. I sometimes mess around with other scenes, and... Yeah, that's about it.


DISCLAIMER: My designs may or may not be crude.

A few other things to know about my designs:

I: Please credit me in the description and/or scene.
II: Because I cannot sign up for Physion, as it is restricted, I will send an image of the requested design, and add tweaks as needed.
III: Schedule:
Mondays - Fridays: 8:14 - 4:30.
Weekends: Not working.

**Reply by nikoniko78** [reply]
Thank you for the information. I will create a scene called Gitney's Things, as I regularly post and don't want my monthly scenes being limited. I can add things you send to me via [email.](https://t.ly/-HCSU) Please note that I am not usually active anymore as I am working on a top secret scene(no spoilers), and studies to work on, so your things might have up to a week's delay before being added. Scripts you create can be added to my repo, just follow the instructions in the Contributing Guidelines under Rules on the readme. If the email link does not work for you, please look at my profile.

**Reply by git900000** [reply]
You're very welcome! And by the way, I have just a little correction: What I was saying was people can request, but we can do both. I will also create a discussion tab for the requests, and I will email images of the designs. People can then go to "Gitney's Things" (Nice name!), and copy my design. Designs will be labeled like this example:


"Thing Name Here"
    -Requested By: 
           User
       
(Design description, written by yours truly ;) )


What do ya think?

**Reply by nikoniko78** [reply]
Sounds good!

**Reply by git900000** [reply]
Another thing: My organization cannot let me email to people outside of it. I apologize. Use my secondary email:

shaweel@outlook.com

I will send you a test email via this one so be on the lookout. It will have the subject "Leo".

**Reply by git900000** [reply]
Sent!

**Reply by git900000** [reply]
Check your spam if you don't see it.

**Reply by nikoniko78** [reply]
Sorry for the late reply. Had to go to a play for school. You should have received an email. Anyways, I am going to add a scene request discussions tab. Feel free to keep me updated there!

**Comment by git900000** [comment]
Right! What was the play?
________________________________

**Reply by nikoniko78** [reply]
some musical about getting into harvard but it had some pretty good songs

**Reply by git900000** [reply]
Nice! My friend loves plays!

**Comment by git900000** [comment]
Wow, this takes me back... :')

**Reply by nikoniko78** [reply]
hii

**Reply by git900000** [reply]
Hey! :D

**Reply by git900000** [reply]
By the way, do you know ssorc j's email? Because I want to ask him if I can join Wilderness Inc.

**Reply by nikoniko78** [reply]
a no b wilderness is a really old roblox branch

**Reply by nikoniko78** [reply]
sorry didmt mean to be rude

**Reply by git900000** [reply]
You're okay! ;)

**Reply by git900000** [reply]
I just found this (Attached).

https://github.com/user-attachments/assets/66a83b7b-f368-4c1a-8ca7-3474520699bf

**Reply by git900000** [reply]
I just created a Chatroom category in Discussions; meet me there!
